### ISSUE TITLE
android.permission.WRITE_EXTERNAL_STORAGE

### ISSUE THREAD

**Issue by krzysztof-swi** [Apparence-io/CamerAwesome#320]
Hi,
This task concerns only Android version:
My app has defined write_external_storage permission in AndroidManifest.xml, as I need it in other place:
<uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />

I use camereawesome for taking pictures, I save them inside application directories, using path_provider.getTemporaryDirectory.
This permission is not required for saving files in this directory.
However when I open the camera, the permission popup appears asking for external storage.
I think the culprit is in CameraPermissions.kt - allPermissions list contains write_external_storage permission and it is never removed.
Maybe asking for this permission could be made optional?

**Comment by MorningGu** [comment]
You are right. I tried to delete the part of the code that contains the write_external_storage permission, so it will no longer request this permission. The reason why the demo does not request permission is because this permission is not registered in the AndroidManifest. After registration, it will still request this permission.
